app/controllers/admin/facilities_controller.rb

```diff
@@ -114,8 +114,9 @@ def validate_at_least_one_facility
 
   def validate_duplicate_rows
     facilities_slice = @facilities.map { |facility|
-      facility.slice(:organization_name, :facility_group_name, :name) }
-    @errors << 'Uploaded file has duplicate facilities' if
+      facility.slice(:organization_name, :facility_group_name, :name)
+    }
```

Comment:
Semantic approach to `{}` vs `do end`, nice. On reading more about this, my favorite reason to do this so far is:

> As Nick Morgan points out below, chaining method calls onto an end looks kinda weird (do ... end.foo)